assets/javascripts/discourse/templates/components/chat-replying-indicator.hbs

```diff
@@ -1,12 +1,10 @@
 <div class="chat-replying-indicator">
-  <span class="replying-text">
-    {{#if shouldDisplay}}
-      <span>{{text}}</span>
-      <span class="wave">
-        <span class="dot">.</span>
-        <span class="dot">.</span>
-        <span class="dot">.</span>
-      </span>
-    {{/if}}
-  </span>
+  {{#if shouldDisplay}}
+    <span class="chat-replying-indicator__text">{{text}}</span>
```

Comment:
This indicator is the kind of thing that could possibly be a simple component in core, would be good to include in the styleguide. (No need to do anything in this PR.)